css/style.css

```diff
@@ -75,6 +75,201 @@ a.logo:hover {
   margin-left: 22px;
 }
 
+.works-container {
+  align-items: center;
+  padding: 14px;
+  margin: 88px 20px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 327px;
+  height: 552px;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+h1.card-title {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  order: 0;
+  flex-grow: 0;
+}
+
+.work-details {
+  display: flex;
+}
+
+.work-item {
+  font-weight: 500;
+  font-size: 13px;
+  padding: 5px;
+  color: #7a869a;
+  line-height: 16px;
+}
+
+.work-writeup {
+  color: #344563;
+  font-weight: 400;
+  font-size: 16px;
+  padding: 5px;
+  line-height: 24px;
+  order: 1;
+  flex-grow: 0;
+}
+
+.counter {
+  border-radius: 30px;
+  width: 8px;
+  height: 8px;
+  margin-top: 7px;
+  background: #c1c7d0;
+}
+
+.work-description {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  padding: 5px;
+  line-height: 16px;
+}
+
+#tool {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 10px;
+  margin: 5px;
+  list-style: none;
+}
+
+.work-button {
+  display: flex;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  text-decoration: none;
+  width: 100px;
+  height: 24px;
+  font-weight: 500;
+  margin: 5px;
+}
+
+.work-tools {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  display: flex;
+}
+
+#projects {
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  border: 1px solid #396df2;
+  width: 120px;
+  padding: 10px 5px;
+  margin: 10px;
+}
+
+.resume-button {
```

Comment:
- [ ] [OPTIONAL] Consider adding hover & active styles for the buttons to improve the UI.